clientapi/routing/joinroom_test.go

```diff
@@ -33,6 +34,10 @@ func TestJoinRoomByIDOrAlias(t *testing.T) {
 
 		cm := sqlutil.NewConnectionManager(processCtx, cfg.Global.DatabaseOptions)
 		caches := caching.NewRistrettoCache(128*1024*1024, time.Hour, caching.DisableMetrics)
+		_, openErr := storage.Open(processCtx.Context(), cm, &cfg.RoomServer.Database, caches)
```

Comment:
This shouldn't be needed, as `roomserver.NewInternalAPI` should already open the storage.